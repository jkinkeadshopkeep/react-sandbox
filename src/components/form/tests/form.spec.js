import React from 'react';
import {render} from '@testing-library/react';
import Form from "../index";
import TextInput from "../../text-input";

describe('Form Component', () => {
    it('should render form with an input', () => {
        const {container} = render(<Form id='hookForm'>
            <TextInput
                type='text'
                name='forename'
                label='Forename'
            />
        </Form>);

        expect(container.firstChild).toMatchSnapshot();
    });
});
