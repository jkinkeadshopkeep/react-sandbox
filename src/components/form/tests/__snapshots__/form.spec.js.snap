// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Form Component should render form with an input 1`] = `
<form
  class="form-box"
  id="hookForm"
  type="button"
>
  <label
    for="forename"
  >
    Forename
  </label>
  <input
    name="forename"
    type="text"
  />
  <br />
</form>
`;
